testsuite/MDAnalysisTests/analysis/test_hole.py

```diff
@@ -55,129 +55,108 @@ def rlimits_missing():
     return False
 
 
-class TestHOLE(TestCase):
+@pytest.mark.skipif(executable_not_found("hole"), reason="Test skipped because HOLE not found")
+class TestHOLE(object):
     filename = PDB_HOLE
-
-    @dec.skipif(executable_not_found("hole"), msg="Test skipped because HOLE not found")
-    def setUp(self):
+    
+    @pytest.fixture()
```

Comment:
so far you made all fixtures in a class static. I was wondering why this isn't necessary here.